spec/services/random_phrase_spec.rb

```diff
@@ -21,5 +21,13 @@
 
       expect(phrase.to_s.length).to eq 24 # 20 chars + 4 spaces
     end
+
+    it 'does not contain the letters I L O U' do
+      100.times do
```

Comment:
perhaps if there is some significant to this number it would be good to name it? I just know that the minute I got in here and some a loop going through 100 times, I'd be tempted to lower it to something like 5. And if doing that means that the test is less effective, it would be great to know why.